generation/day_ahead.py

```diff
@@ -48,16 +48,28 @@
 
 
 @entsoe_data_bp.cli.command("import-day-ahead-generation")
+@click.option(
+    "--from-date",
+    required=False,
+    type=click.DateTime(["%Y-%m-%d"]),
+    help="Query data from this date onwards. If not specified, defaults to today",
+)
+@click.option(
+    "--to-date",
+    required=False,
+type=click.DateTime(["%Y-%m-%d"]),
+    help="Query data until this date (inclusive). If not specified, defaults to tomorrow.",
+)
 @click.option(
     "--dryrun/--no-dryrun",
     default=False,
     help="In Dry run, do not save the data to the db.",
 )
 @with_appcontext
 @task_with_status_report
-def import_day_ahead_generation(dryrun: bool = False):
+def import_day_ahead_generation(dryrun: bool = False, from_date: Optional[datetime] = None, to_date: Optional[datetime] = None):
     """
-    Import forecasted generation for the upcoming day.
+    Import forecasted generation for any date range, defaulting to today and tomorrow.
```

Comment:
Why does this PR change the default from tomorrow to today&tomorrow?

Reply:
Actually, I'm okay with keeping the default to just query data for tomorrow. I had changed the default mainly because I was testing the code in the morning, when tomorrow's data wasn't available yet.